superdesk/default_settings.py

```diff
@@ -364,7 +367,7 @@ def local_to_utc_hour(hour):
 ]
 
 #: Specify what modules should be enabled
-INSTALLED_APPS = []
+INSTALLED_APPS = ['superdesk.sams']
```

Comment:
Are we going to have SAMS enabled by default?

Reply:
Missed it, have removed 'superdesk.sams' from INSTALLED_APPS. If sams is enabled this will be added in settings.py at superdesk/server in customer repository.